.chloggen/hecexp-fix-max-event.yaml

```diff
@@ -0,0 +1,11 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: prometheusreceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Make sure the `max_event_size` option is used to drop events larger that that instead of using it for batch size.
```

Comment:
```suggestion
note: Make sure the `max_event_size` option is used to drop events larger than `max_event_size` instead of using it for batch size.
```

Reply:
Thanks